tests/test_dcgm_monitor.py

```diff
@@ -49,11 +50,14 @@ def test_record_memory(self):
         # Assert instance types
         for record in records:
             self.assertIsInstance(record.device(), GPUDevice)
+            self.assertIsInstance(record.value(), float)
+            self.assertIsInstance(record.timestamp(), int)
```

Comment:
Wondering if these tests should be done here or in a different test that just tests the constructors of all Records. Looks good for now.